index.html

```diff
@@ -9,7 +9,7 @@
     <meta property="og:title" content="Crypto Duckies Migration" />
     <title>Crypto Duckies Migration</title>
 
-    <link rel="stylesheet" type="text/css" href="/dist/css/main.css">
+    <link rel="stylesheet" type="text/css" href=" dist/css/main.css">
```

Comment:
Just to get it working with github pages hosting, not a real change.
Test site: https://octal-crypto.github.io/crypto-duckies-migration/